app/stores/AuthStore.ts

```diff
@@ -68,19 +68,21 @@ export default class AuthStore {
 
     try {
       data = JSON.parse(localStorage.getItem(AUTH_STORE) || "{}");
-    } catch (_) {
-      // no-op Safari private mode
+    } catch (err) {
+      Sentry.captureException(err);
     }
 
     this.rehydrate(data);
+
     // persists this entire store to localstorage whenever any keys are changed
     autorun(() => {
       try {
         localStorage.setItem(AUTH_STORE, this.asJson);
-      } catch (_) {
-        // no-op Safari private mode
```

Comment:
This used to be the case, but I just tested and it's no longer an issue on desktop. Will capture some exceptions in production and see if we can ignore something more specific as this catch-all is what resulted in the regression going unnoticed here